packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -193,7 +275,7 @@ const errorMap = {
   // Plugin errors
   "11321": {
     text: context =>
-      `"${context.pluginName}" threw an error while running the ${context.api} lifecycle:\n\n${context.message}`,
+      `"${context.pluginName}" threw an error while running the ${context.api} lifecycle:\n\n${context.message}\n\n${context.codeFrame}`,
```

Comment:
We might not always have `codeFrame` - for conditional maybe something like this:
```suggestion
      [ 
        `"${context.pluginName}" threw an error while running the ${context.api} lifecycle:\n\n${context.message}`,
        context.codeFrame,
      ].filter(Boolean).join(`\n\n`)
```

to avoid printing empty lines if `codeFrame` is not available?

(this potentially applies to other errors as well)

Reply:
Ah, good catch. We should also use `optionalGraphQLInfo()` here then (see top of file).